packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -719,6 +750,12 @@ class SliverGeometry with Diagnosticable {
   ///  * [RenderViewport.cacheExtent] for a description of a viewport's cache area.
   final double cacheExtent;
 
+  /// The amount of space allocated to the cross axis.
+  ///
+  /// This value should equal the [crossAxisExtent] of the sliver. If unset/null, then
+  /// the cross axis extent of the sliver is the same as the [SliverConstraints.crossAxisExtent].
```

Comment:
```suggestion
  /// This value should equal the [crossAxisExtent] of the sliver. If null, then
  /// the cross axis extent of the sliver is assumed to be the same as the
  /// [SliverConstraints.crossAxisExtent]. This is because slivers typically
  /// consume all of the extent that is available in the cross axis.
  ///
  /// See also:
   ... add * to the cross axis extent sliver - since it is the exception to the rule
```

Maybe also add reference to the cross axis group and corss axis expanded?